slownie.py

```diff
@@ -91,7 +91,9 @@ def slownie999(i):
 def slownie(i, jeden=False, unit=None):
     i = int(i)
     if i < 0:
-        raise ValueError('i < 0')
+        i = i * -1
+        return u' '.join(["minus", slownie(i, jeden, unit)])
+        # raise ValueError('i < 0')
```

Comment:
No commented out code, please. That's just distraction for people reading the code, and even greater distraction for people reading the PR diff.